scripts/ffmpeg-config

```diff
@@ -46,4 +46,7 @@ echo Using ffmpeg options: $OPTIONS "$@"
 
 mkdir -p "$BUILD"/ffmpeg_build
 cd "$BUILD"/ffmpeg_build
+# need to link against stdc++ in case libplacebo was built with glslang,
+# which requires that
+export LDFLAGS="$LDFLAGS -lstdc++"
```

Comment:
this should happen automatically though, did you check whether libplacebo.pc references glslang.pc and that then links stdc++?
if it doesn't work anyway (we had this case before) then `$(pkg-config --libs glslang)` or similar would still be a more portable way to get this to work

Reply:
Ok I checked and glslang has no pkg-config file, only cmake integration.
It doesn't matter too much but the portability issue I saw here is that there is no guarantee the C++ standard libray is called `stdc++` ([libc++](https://libcxx.llvm.org/) also exists).

Reply:
Shouldn't we put the link flag into the libplacebo pc? (And, actually, doesn't libplacebo already?)

Reply:
Having it in libplacebo might be better, the things that link against it (like ffmpeg and mpv in this case) would know about it I guess?

At the moment libplacebo does not have it in its pc; with libunwind and glslang enabled, `libplacebo.pc` has only `Libs: -L${libdir} -lplacebo -lm -pthread -ldl -lglslang -lOSDependent -lHLSL -lOGLCompiler -lSPVRemapper -lSPIRV -lSPIRV-Tools-opt -lSPIRV-Tools -lpthread`.
Tried both libplacebo release and master.

Reply:
Oh, indeed - I was confusing it with the `pthread` entry, which we also had to add to the pc file manually

Reply:
I think it would be reasonable for libplacebo to add the c++ standard library to its explicit deps if glslang is used, *if* that can be done portably.

Reply:
Isn't it my understanding that `libstdc++` should *not* be added when linking into a C++ program?

Also, I'm not yet sure how to find the c++ standard library in a portable way using meson.

Reply:
I guess if you use g++ to link, this isn't a problem. The whole issue is that we're doing a hybrid static / shared build which leads to problems like this for which there is no good solution. 

I think the best we can do is have the `mpv-build` scripts patch the pkg-config file for `libplacebo` (and / or shaderc if we add it) to add `-lstdc++`. if someone could devise a trick to universally detect the proper c++ library, that would be a bonus.

Reply:
In any case the current workaround is _good enough_.